uber/models.py

```diff
@@ -1151,7 +1156,7 @@ def ribbon_and_or_badge(self):
 
     @property
     def is_dealer(self):
-        return bool(self.tables and self.tables != '0' and (not self.registered or self.amount_paid or self.cost))
+        return bool(self.tables and self.tables != '0' and self.tables != '0.0' and (not self.registered or self.amount_paid or self.cost))
```

Comment:
For whatever reason, `Group.tables` was declared as a `Numeric` instead of an `Integer`. This means that "0.0" is a valid value.